src/styles/global.css

```diff
@@ -1,12 +1,12 @@
 @font-face {
   font-family: 'Hind Madurai';
-  src: url("/desmos/fonts/HindMadurai-Regular.ttf");
+  src: url("/persistence/fonts/HindMadurai-Regular.ttf");
   font-weight: 400;
 }
 
 @font-face {
   font-family: 'Hind Madurai';
-  src: url("/desmos/HindMadurai-Regular.ttf");
+  src: url("/persistence/HindMadurai-Regular.ttf");
```

Comment:
Can you add  `fonts` after `persistence` to make line.9 the same as line.3? Sorry it's due to a bug from latest base release. I'll fix it soon so doesn't affect new bds.